package.json

```diff
@@ -179,6 +179,7 @@
     "babel-jest": "^27.5.1",
     "babel-loader": "^8.2.3",
     "brotli-webpack-plugin": "^1.1.0",
+    "buffer": "^6.0.3",
```

Comment:
when are we using buffer?

Reply:
getting "buffer is not defined" when developing locally with npm link across, a quick check points to webpack 5 compatibility issues 🤷‍♀️

https://stackoverflow.com/questions/68707553/uncaught-referenceerror-buffer-is-not-defined <- this worked